app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -16,6 +16,7 @@ def call
 
         send_welcome_notification unless notification_enqueued
         send_authentication_notification unless notification_enqueued
+        send_customize_notification unless notification_enqueued
```

Comment:
Since we have experience customization and feed customization, I think we'll have to rename this to something more explicit. Perhaps `send_customize_ux_notification`?

Reply:
Hah, I actually _did_ have it as `send_customize_ux_notification` but it felt super long, which is why I shortened it :) Can change it back if that makes more sense, though!

Reply:
Agreed with @maestromac . I know that what I'm about to suggest is a little verbose, but I think it follows the naming convention above a bit better; how about `send_ux_customization_notification`?